test/Microsoft.AspNetCore.Server.KestrelTests/FrameResponseStreamTests.cs

```diff
@@ -61,6 +61,9 @@ public void BeginReadThrows()
             var stream = new FrameResponseStream(new MockFrameControl());
             Assert.Throws<NotSupportedException>(() => stream.BeginRead(new byte[1], 0, 1, null, null));
         }
+#elif NETCOREAPP2_0
```

Comment:
We're standardizing across this in all the projects. I think it'll be overkill to specify this eveywhere we use this.